### PR TITLE
Create stra pia requirements for app teams

### PR DESCRIPTION
**Pull request by NickCorcoran**
Introduced changes to devops security considerations for the following:
- created a section for STRA/PIA requirements for apps
- Updated links from cloud. to digial. sites
- Other minor text changes

### CODE REVIEW

**Comment by Pilargit12** [comment]
@NickCorcoran  let's wait for @StevenBarre and @w8896699 last review before merging to main, thank you!

File: src/docs/security-and-privacy-compliance/devops-security-considerations.md
Comment (StevenBarre) [inline src/docs/security-and-privacy-compliance/devops-security-considerations.md:273]: Unsure what EKS is? Is that supposed to be EFK for ElasticSearch/FluentD/Kibana ?

File: src/docs/security-and-privacy-compliance/devops-security-considerations.md
Comment (StevenBarre) [inline src/docs/security-and-privacy-compliance/devops-security-considerations.md:359]: I think this is a setting at the storage class level, not the pvc level. I don't think app teams currently can create an encrypted PVC.

File: src/docs/security-and-privacy-compliance/devops-security-considerations.md
Comment (StevenBarre) [inline src/docs/security-and-privacy-compliance/devops-security-considerations.md:361]: Every clusters storage is on its own VLAN segment that is not route. Not just Emerald.

**Review by StevenBarre — CHANGES_REQUESTED** [review]

**Review by w8896699 — APPROVED** [review]
Looks good to me!

File: src/docs/security-and-privacy-compliance/devops-security-considerations.md
Comment (NickCorcoran) [reply src/docs/security-and-privacy-compliance/devops-security-considerations.md:359]: This was based on a convo we had and testing in the lab:  https://app.zenhub.com/workspaces/platform-experience-5bb7c5ab4b5806bc2beb9d15/issues/gh/bcdevops/developer-experience/1970.   If I got wording wrong I can change.

File: src/docs/security-and-privacy-compliance/devops-security-considerations.md
Comment (NickCorcoran) [reply src/docs/security-and-privacy-compliance/devops-security-considerations.md]: Ajusted.

File: src/docs/security-and-privacy-compliance/devops-security-considerations.md
Comment (NickCorcoran) [reply src/docs/security-and-privacy-compliance/devops-security-considerations.md]: Updated.  Thanks.

File: src/docs/security-and-privacy-compliance/devops-security-considerations.md
Comment (StevenBarre) [reply src/docs/security-and-privacy-compliance/devops-security-considerations.md:359]: Ya, we tested an encrypted storage class in the lab, but never rolled it out to prod. I think there wasnt much demand for it?

It would be a different storage class you'd request when creating a pvc, instead of `netapp-file-standard` youd pick `netapp-file-encrypted`.

**Review by StevenBarre — APPROVED** [review]

**Pull request merged**
